src/GraphQL/Execution/ExecutionError.cs

```diff
@@ -98,7 +78,7 @@ private void SetData(IDictionary dict)
             }
         }
 
-        private static string GetErrorCode(Exception exception) => _exceptionErrorCodes.GetOrAdd(exception.GetType(), NormalizeErrorCode);
+        public static string GetErrorCode(Exception exception) => _exceptionErrorCodes.GetOrAdd(exception.GetType(), NormalizeErrorCode);
```

Comment:
No need to change this line, only `HasCodes` / `Codes` access modifiers to public.

Reply:
If we change those to public, it would change the Serialization of the class.

Reply:
Ignore these properties in your custom contract resolver. Use `ShouldSerialize`.

Reply:
Actually I was wrong, the ExecutionError already uses a special json converter, so making these public didn't affect serialization.  This should be changed now.

Reply:
Indeed, I myself forgot about it.